src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -52,16 +52,17 @@
   <Target Name="SetNuSpecProperties" Condition=" Exists('$(NuspecFile)') ">
     <PropertyGroup>
       <NuspecProperties>
-        id=$(PackageId);
-        description=$(PackageDescription);
-        version=$(PackageVersion);
-        authors=$(Authors);
-        copyright=$(Copyright);
-        projecturl=$(PackageProjectUrl);
-        tags=$(PackageTags);
-        configuration=$(Configuration);
-        tfm=$(TargetFramework);
-        packagetype=Dacpac
+          id=$(PackageId);
+          description=$(PackageDescription);
+          version=$(PackageVersion);
+          authors=$(Authors);
+          copyright=$(Copyright);
+          projecturl=$(PackageProjectUrl);
+          tags=$(PackageTags);
+          configuration=$(Configuration);
+          tfm=$(TargetFramework);
+          packagetype=Dacpac;
+          $(NuspecProperties)
```

Comment:
This change allows for additional properties to be passed in when using the nuspec file.

Reply:
I face the same issue (custom `<NuspecProperties>` from csproj file being ignored) and reached the same conclusion. I can confirm this fixes the issue.